Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
```

Comment:
The problem with using the GUI code as is here, in tests, is that it depends on multiple other libraries/classes that are part of the GUI code base (e.g. to support undo or to get/manipulate nodes from Navigator or to communicate with the Visualization). Pulling these and their dependencies into a test is a massive undertaking and may not be possible, in general, without including the netbeans platform itself, sorry if that wasn't made clear earlier.
We can discuss whether this belongs in the OpenSimContext class in person.